.github/workflows/win_python_matrix_test.yml

```diff
@@ -0,0 +1,85 @@
+name: Windows Pyomo Tests Workflow (Slim, Matrixed, Nightly)
+
+on: 
+  pull_request: 
+    branches:
+      - master
+
+jobs:
+  pyomo-tests:
+    name: Pyomo Test (${{ matrix.python-version }}, ${{ matrix.os }})
```

Comment:
Similar to #1232, can we change this to just `py${{ matrix.python-version }}`?

Reply:
Has been changed.